merge_and_rename_NGI_fastq_files.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env/python
+# Merges all fastq_files from each samples into one file. 
+# ARGV 1 Base directory with files to be merged
+# ARGV 2 Destination dir of the merged output files
+
+import re
+import os
+import sys
+import shutil
+def merge_files(dest_dir, fastq_files):
+    #Match NGI sample number from flowcell
+    sample_pattern=re.compile("^(.+)_S[0-9]+_.+_R([1-2])_")
+    #Remove files that already have the right name (i.e have been merged already)
+    matches=[]
+    for fastq in fastq_files:
+        try:
+            match=sample_pattern.search(fastq).group(1)
+            if match:
```

Comment:
if it does not match it is better to warn the user, we expect to always match the fastq.gz files